source/guide_buildbot.rst

```diff
@@ -0,0 +1,266 @@
+.. highlight:: console
+
+.. author:: Thomas Hoffmann <uberlab@emptyweb.de>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/buildbot.svg
+      :align: center
+
+########
+Buildbot
+########
+
+Buildbot is an open-source framework for automating software build, test, and release processes. At its core, Buildbot is a job scheduling system: it queues jobs, executes the jobs when the required resources are available, and reports the results. It can be easily installed and serve as a continuous integration platform to be used together with a variety of version control solutions, including gitea.
+
+In this tutorial, we will first follow along with the official installation manual and set up a ``hello world``-system and will extend the standard installation of Buildbot_ with a gitea_-Plugin.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Python_
+  * git_
+  * supervisord_
+  * ssh port forwarding
+  * Folder/File Permissions
+
+.. note:: Recommended reading to follow along and go beyond this guide:
+
+  * `Official Buildbot Manual <https://docs.buildbot.net/latest/manual/index.html>`_
+  * `Official Buildbot Tutorial <https://docs.buildbot.net/latest/tutorial/index.html>`_
+  * `Buildbot in 5 Minutes User Tutorial <https://docs.buildbot.net/latest/tutorial/fiveminutes.html>`_ (warning: based on an older version of buildbot! Code examples should be taken with a grain of salt. It explains the mechanics quite nicely, though)
+
+License
+=======
+
+Buildbot is released under the GNU General Public License v2.0
+
+  * https://www.gnu.org/licenses/old-licenses/gpl-2.0
+
+Prerequisites
+=============
+
+For ease of use, you should set your default Python_ version to 3.6 and upgrade the ``pip3.6`` installation as well:
+
+::
+
+ [isabell@stardust ~]$ ln -s /usr/bin/python3.6 $HOME/bin/python
```

Comment:
I agree as well - same as above with pip, I did this step in "vorauseilendem Gehorsam" since I got burned by missing out on updating/linking required software before ^^ for the sake of simplicity I should maybe get used to checking reduction of steps as well.